psi4/driver/procrouting/proc.py

```diff
@@ -1299,6 +1300,16 @@ def scf_helper(name, post_scf=True, **kwargs):
     if ref_wfn is not None:
         raise ValidationError("Cannot seed an SCF calculation with a reference wavefunction ('ref_wfn' kwarg).")
 
+    # decide if we keep the checkpoint file
+    _chkfile = kwargs.get('write_orbitals', True)
+    write_checkpoint_file = False
+    if isinstance(_chkfile, str):
+        write_checkpoint_file = True
+        filename = kwargs.get('write_orbitals')
```

Comment:
I wonder if the kwarg should be `orbitals_write` so that we don't have kwarg `write_orbitals` and keyword `orbitals_write`.

Reply:
That is a good point. The whole`<file>_write` "convention" just feels a bit awkward to me to be honest.

It would change the old `write_orbitals` keyword without deprecation warning, though.
Though I think the kwarg was never documented so maybe that is okay.

Reply:
> That is a good point. The whole<file>_write "convention" just feels a bit awkward to me to be honest.

Agreed. I don't remember quite why we settled on verb at the end for `HESSIAN_WRITE` and the like, but I think it's worth keeping to the pattern for keywords.

> It would change the old write_orbitals keyword without deprecation warning, though.
Though I think the kwarg was never documented so maybe that is okay.

Ah, ok. Probably better to keep it as-is then. Sorry to have created the orbitals_write/write_orbitals inconsistency for you, then.